spec/setup_acceptance_node.pp

```diff
@@ -23,22 +23,22 @@
 # $baseurl = "https://fedorapeople.org/groups/katello/releases/yum/nightly/pulpcore/el${major}/x86_64/"
 $baseurl = "http://koji.katello.org/releases/yum/katello-nightly/pulpcore/el${major}/x86_64/"
 
-file { 'pulpcore-repo-extra-config':
-  path    => '/etc/yum/pulpcore.conf',
-  content => 'module_hotfixes=1',
-  before  => Yumrepo['pulpcore'],
-}
+$pulpcore_repo_conf = @("CONF")
+[pulpcore]
+baseurl=${baseurl}
+gpgcheck=0
+module_hotfixes=1
+CONF
 
-yumrepo { 'pulpcore':
-  baseurl  => $baseurl,
-  gpgcheck => 0,
-  include  => 'file:///etc/yum/pulpcore.conf',
+file { '/etc/yum.repos.d/pulpcore.repo':
+  ensure  => 'present',
```

Comment:
In this case it doesn't matter that much, but I've learned to be explicit with the `file` type and state whether it's a file or directory.
```suggestion
  ensure  => 'file',
```